tcadmin/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+
+
+ACTION_CREATE = "create"
+ACTION_UPDATE = "update"
+ACTION_DELETE = "delete"
+ACTIONS = [ACTION_CREATE, ACTION_UPDATE, ACTION_DELETE]
+
+BEFORE_APPLY = "before_apply"
+AFTER_APPLY = "after_apply"
```

Comment:
I recall having difficulty with circular imports when putting useful symbols at the package root.  Since these aren't in the public API, could they be defined in `tcadmin.constants` or the like?

Reply:
Done.